extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -76,19 +77,26 @@ const getMetadata = (player, props) =>
 function VideoWrapperWithRef(
   {
     component: Component = 'video',
```

Comment:
Why is `component` preferred to `as`?

Reply:
I think the reason @alanorozco did this is because `as` normally means "use this component as a root", but in this case the video component is embedded inside the player.